client/client_persistence_test.go

```diff
@@ -15,48 +15,22 @@
 package client_test
 
 import (
-	"math/big"
 	"math/rand"
 	"testing"
 
-	"perun.network/go-perun/apps/payment"
 	chprtest "perun.network/go-perun/channel/persistence/test"
-	chtest "perun.network/go-perun/channel/test"
-	"perun.network/go-perun/client"
 	ctest "perun.network/go-perun/client/test"
-	"perun.network/go-perun/pkg/test"
-	"perun.network/go-perun/wire"
 )
 
 func TestPersistencePetraRobert(t *testing.T) {
-	rng := test.Prng(t)
-	for i := 0; i < 2; i++ {
-		setups := NewSetupsPersistence(t, rng, []string{"Petra", "Robert"})
-		roles := [2]ctest.Executer{
+	runTwoPartyTest(t, func(rng *rand.Rand) (setups []ctest.RoleSetup, roles [2]ctest.Executer) {
```

Comment:
Nice!

Reply:
The linter was suddenly complaining about duplicate code...